tutorial/README.md

```diff
@@ -0,0 +1,99 @@
+# Tutorial: DeepSparse Server Monitoring via Prometheus/Grafana
+
+One of the features of the [DeepSparse Server](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server) is its compatibility with the monitoring services popular among ML practitioners. 
+
+This tutorial will show you how to monitor the DeepSparse server using the Prometheus/Grafana stack.
+You will learn how to quickly configure DeepSparse Server and Prometheus, to continuously and seamlessly monitor the Server.
+
+In the nutshell: once configured, the Server exposes `metrics` endpoint, which in turn is scrapped by Prometheus - different logs are being collected, aggregated, and stored. 
+Prometheus allows external entities (like Grafana) to access these aggregated logs via the query language PromQL. 
+The role of Grafana is to allow users to visualize these metrics by creating dashboards.
+
+## Prerequisites
+### Structure of the Repository
+The repository has the following structure:
+
+```bash
+.
+├── client 
+│   ├── client.py # simple client application
+│   └── piglet.jpg 
+├── deepsparse_server_config.yaml # specifies the configuration of the DeepSparse server
+├── demo.sh # script that runs the contents of this tutorial
+├── docker # specifies the configuration of the containerized Prometheus/Grafana stack
+│   ├── docker-compose.yaml
+│   └── prometheus.yaml
+└── grafana # specifies the design of the Grafana dashboard
+    └── dashboard.json
+```
+### Installing DeepSparse Server
+We recommend you to install deepsparse server inside the virtual environment:
+
+```bash
+pip install virtualenv
+virtualenv deepsparse_venv
+
+source {$virtual_env_path}/bin/activate
+pip install deepsparse[server]
+```
+
+### Running the Instructions of this README
```

Comment:
The script was very useful for me for bugging purposes and this is why I kept it. Feels good to have the contents of the tutorial condensed into one command. 
This script was written for the first time for the other tutorial (Server + K8 + Prometheus/Grafana) because this tutorial is more complex. However, in the context of this tutorial, I think I am fine with removing the bash script.